conf/routes

```diff
@@ -28,9 +28,13 @@ PUT     /api2/atom/:id/pluto-id         controllers.Api2.setPlutoId(id)
 PUT     /api2/pluto/:id                 controllers.Api2.sendToPluto(id)
 GET     /api2/pluto                     controllers.Api2.getPlutoAtoms()
 
-
 DELETE  /api2/atom/:id                  controllers.Api2.deleteAtom(id)
 
+GET     /api2/pluto/projects            controllers.PlutoProjectController.listProjects()
+GET     /api2/pluto/projects:id         controllers.PlutoProjectController.getProject(id)
```

Comment:
hmm, good point! I've been curling it locally with no problems... not sure how though!